src/main/java/gift/exception/ErrorResponse.java

```diff
@@ -0,0 +1,54 @@
+package gift.exception;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ErrorResponse {
+
+  private String message;
+  private int status;
+  private String code;
+  private List<FieldError> errors;
+
+  // 에러코드 객체만 들어왔다면 그 객체를 이용해서 에러응답객체를
+  private ErrorResponse(final ErrorCode errorCode) {
+    this.message = errorCode.getMessage();
+    this.status = errorCode.getStatus().value();
+    this.code = errorCode.getCode();
+    this.errors = new ArrayList<>();
+  }
+
+  // 에러코드 객체와, 필드에러 리스트가 들어왔을때 응답객체 생성
+  private ErrorResponse(final ErrorCode errorCode, final List<FieldError> errors) {
+    this.message = errorCode.getMessage();
+    this.status = errorCode.getStatus().value();
+    this.code = errorCode.getCode();
+    this.errors = errors;
+  }
```

Comment:
```suggestion
  private ErrorResponse(final ErrorCode errorCode) {
    this(errorCode, new ArrayList<>());
  }

  // 에러코드 객체와, 필드에러 리스트가 들어왔을때 응답객체 생성
  private ErrorResponse(final ErrorCode errorCode, final List<FieldError> errors) {
    this.message = errorCode.getMessage();
    this.status = errorCode.getStatus().value();
    this.code = errorCode.getCode();
    this.errors = errors;
  }
```
아래쪽 생성자가 위쪽 생성자를 포함할 수 있지 않을까요? :)